packages/dds/tree/README.md

```diff
@@ -151,7 +151,20 @@ When it is more designed, some details for how it works belong in this section (
 
 ### Editing
 
-TODO: add a diagram for this section.
+Edit related data flow with solid arrows.
+Key view related updates made in response with dotted arrows.
+
+This shows editing during a transaction:
+
+```mermaid
+graph RL;
```

Comment:
Was this (and the one below) meant to be LR? It's a bit hard to read in the data flow sense but I can also see how RL makes sense when it comes to editing back in a pipe. I prefer LR for this though since it isn't a pipe.

Reply:
This was on purpose. I mostly wanted the components to be in roughly the same places in all the data flow diagrams, and in this one the data flows mainly in the reverse direction.

Reply:
Oh I see. Using [subgraphs](https://mermaid-js.github.io/mermaid/#/flowchart?id=subgraphs) might help make this more clear if you keep some consistent groups of components throughout the diagrams. Feel free to resolve this though.

Reply:
I added some subgraphs, which I do think help. Good suggestion. They also broke the right to left setup, but I think left to right is fine now.